### ISSUE TITLE
Errors on rendering

### ISSUE THREAD

**Issue by eeholmes** [quarto-journals/elsevier#1]
I tried to render from the template (no changes) using the instructions.

Quarto version 0.9.653

```
quarto use template quarto-journals/elsevier
quarto render article.qmd --to elsevier-pdf
```

returned this error
```
Error running filter _extensions/elsevier/elsevier.lua:
_extensions/elsevier/elsevier.lua:60: attempt to call a nil value (field 'citeMethod')
stack traceback:
```

I can render the other journal themes. Just not this one.

Note, I quickly figured this out, but `quarto use template quarto-journals/elsevier` did not create `article.qmd`. I named my dir `test` and it created `test.qmd`. So I needed to run `quarto render test.qmd --to elsevier-pdf`

**Comment by cathblatter** [comment]
I am observing the same behaviour with Quarto version 1.0.3

**Comment by dragonstyle** [comment]
Can you please upgrade to the latest version of quarto and see whether this issue persists? (V1.0.37)

**Comment by eeholmes** [comment]
Seems fixed with v1.0.37!
